ilastik/applets/featureSelection/__init__.py

```diff
@@ -20,4 +18,44 @@
 # on the ilastik web site at:
 # 		   http://ilastik.org/license.html
 ###############################################################################
+from typing import Callable, Iterable, Union
+
+from ilastik.applets.base.applet import DatasetConstraintError
+
+_SCALES = Iterable[Union[int, float]]
+
+
+class FeatureSelectionConstraintError(DatasetConstraintError):
+    def __init__(
+        self,
+        appletName: str,
+        invalid_scales: _SCALES,
+        invalid_z_scales: _SCALES,
+        fixing_dialogs=Iterable[Callable],
```

Comment:
Default arguments for a seemingly common case:
```suggestion
        invalid_z_scales: _SCALES = (),
        fixing_dialogs=Iterable[Callable] = (),
```